process/api.py

```diff
@@ -87,6 +99,161 @@ class CollectionViewSet(viewsets.ViewSetMixin, ListAPIView):
         "completed_at",
     ]
 
+    @csrf_exempt
```

Comment:
Strange, pelican-frontend doesn't use this and yet it works?

Reply:
I think maybe we need to review the inheritance. Pelican inherits from `(mixins.RetrieveModelMixin, viewsets.GenericViewSet)`, `(viewsets.ViewSet)`, or `(views.APIView)`. Process inherits from `(viewsets.ViewSetMixin, ListAPIView)` or `(viewsets.ViewSetMixin, RetrieveAPIView)`. 

`ViewSetMixin` is not mentioned in the docs, and should probably not be used.